pkg/services/blockfetcher/blockfetcher.go

```diff
@@ -189,6 +228,68 @@ func getHeaderSizeMap(chain config.Blockchain) map[int]int {
 	return headerSizeMap
 }
 
+func (bfs *Service) getContainerHeight() (uint32, error) {
+	startIndex := bfs.heightFunc()/bfs.cfg.IndexFileSize + 1
+	for {
+		prm := client.PrmObjectSearch{}
+		filters := object.NewSearchFilters()
+		filters.AddFilter(bfs.cfg.IndexFileAttribute, fmt.Sprintf("%d", startIndex), object.MatchStringEqual)
+		filters.AddFilter("IndexSize", fmt.Sprintf("%d", bfs.cfg.IndexFileSize), object.MatchStringEqual)
+		prm.SetFilters(filters)
+
+		ctx, cancel := context.WithTimeout(bfs.ctx, bfs.cfg.Timeout)
+		blockOidsObject, err := bfs.objectSearch(ctx, prm)
```

Comment:
After PR finalisation it will be useful to add a comment about this code to https://github.com/nspcc-dev/neo-go/issues/3744, because it will be affected by index files support removal.